docs/source/examples/annotations.rst

```diff
@@ -28,8 +27,23 @@ Annotations
             tip_text = Text('(2, 2)').next_to(arrow.get_end(), RIGHT)
             self.add(numberplane, dot, arrow, origin_text, tip_text)
 
+.. manim:: UsefulAnnotations
+    :save_last_frame:
+
+    class UsefulAnnotations(Scene):
+        def construct(self):
+            m0 = SmallDot()
+            m1 = AnnotationDot()
+            m2 = LabeledDot("ii")
+            m3 = LabeledDot(MathTex(r"\alpha").set_color(ORANGE))
+            m4 = CurvedArrow(ORIGIN, 2*LEFT)
+            m5 = CurvedDoubleArrow(ORIGIN, 2*RIGHT)
+
+            self.add(m0, m1, m2, m3, m4, m5)
+            for i, mobj in enumerate(self.mobjects):
+                mobj.shift(DOWN * (i-3))
```

Comment:
Why is this example here?
It is strange because it is already in master

Reply:
Seems a bunch of commits got mixed into this PR from rebasing upstream/master onto the fork's master

Reply:
I think I found an answer here:
>This happens with GitHub when you squash commits merged in from the target branch.
I had been using squash and merge with Github as the default merge strategy, including merges from the target branch. This introduces a new commit and GitHub doesn't recognize that this squashed commit is the same as the ones already in master (but with different hashes). Git handles it properly but you see all the changes again in GitHub, making it annoying to review. The solution is to do a regular merge of these pulled in upstream commits instead of a squash and merge. When you want to merge in another branch into yours as a dependency, git merge --squash and revert that single commit before pulling from master once that other branch has actually made it to master.
EDIT: another solution is to rebase and force push. Clean but rewritten history



from https://stackoverflow.com/questions/16306012/github-pull-request-showing-commits-that-are-already-in-target-branch
so I think this is nothing we should worry about.

Reply:
I didn't use merge --squash specifically, but when I did ``git rebase -i`` I definitely chose to squash commits as they came in (and there was 9 actions taken in the rebase with a couple of merge conflicts so it happened a few times). I'll force push to make it easier to review. In the future, I think I'll just use a feature branch instead of developing on master though

Reply:
Also in the link you replied there's a solution I'll try now (changing the base twice), but the link sample they posted in the top answer[ also works](https://github.com/ManimCommunity/manim/compare/ManimCommunity:master...jsonvillanueva:master)